src/components/drag-and-drop/draggable-context/draggable-context.js

```diff
@@ -142,11 +152,23 @@ class DraggableContext extends React.Component {
     this.setState({ activeIndex: null });
   }
 
+  customDragLayer = () => {
+    if (this.props.customDragLayer) {
+      return this.props.customDragLayer;
+    }
+    return <CustomDragLayer />;
```

Comment:
As this is not adding any additional props could it just be defined as a default prop removing the need for this method. Not sure exactly how this works for components